packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -15,10 +15,13 @@ This guide explains how to customize your Okta org with your URL domain name and
 
 **What you need**
 
-For customizing your Okta URL domain:
+For customizing an Okta-managed domain or your own domain:
```

Comment:
In both cases, the domain is the customers.  The difference between the two is that the certificate is managed by Okta or provided by the customer.  Not sure how to word this but as-is it looks like ownership of the domain is the difference and that isn't accurate.